### PR TITLE
Add dice game

### LINKED ISSUE #64: Dice Rolling Simulator
**Describe the project you want to add with tech stack**
HTML/CSS and JavaScript

**Expected behavior**
A simple project which will ask the user if he or she wants to roll the dice or not. If yes, then the computer will roll a random dice and project its output.

**Additional context**
A program which can be used in a fun way to play dice game with your friends (especially if you doubt the physical dice used )

### PR DESCRIPTION
**Pull request by Subha-5**
## Description
Adding a dice game to project folder for Hacktoberfest 2022

Close #64 

## Dependencies
No external dependencies

## Future Improvements
The game can be made more user interactive based on the feedbacks by the user.

## Screenshots of relevant screens
Screenshot from 2022-10-13 23-54-14

Screenshot from 2022-10-13 23-53-37

## Developer's checklist

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-check on my work

**If changes are made in the code:**

- [ ] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [ ] My changes in code generate no new warnings
- [ ] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [ ] I have added relevant screenshots in my PR

### CODE REVIEW

**Comment by Subha-5** [comment]
@Abbhiishek this will close #64

**Comment by Abbhiishek** [comment]
- Consider the changes written in code :)
https://webners-git-fork-subha-5-new-branch-abbhiishek.vercel.app/

If you face some issue do let me know 😃

File: CONTRIBUTORS.json [hunk 1]
```diff
@@ -293,5 +293,37 @@
...
+            "Open-source",
+            "App Development",
+            "Software Development"
+        ],
+        "Projects": [
+            {
+                "id": 1,
+                "name": "Webners",
+                "description": "Dice Simulator",
+                "link": "https://github.com/Subha-5/Webners"
```
Comment (Abbhiishek) [inline CONTRIBUTORS.json:316]: This link must be something like this :
```
https://github.com/Abbhiishek/Webners/tree/main/Contributions/Dice Roll game
```
When your commits are merged this link will then work and land you to your code base in Webners Project.

Comment (Abbhiishek) [inline CONTRIBUTORS.json:316]: This link must be something like this :
```
https://github.com/Abbhiishek/Webners/tree/main/Contributions/Dice Roll game
```
When your commits are merged this link will then work and land you to your code base in Webners Project.

File: Contributions/Dice Roll game/README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Your Project Name
+Double Dice Simulator
+
+### Description
+
+A simple project which will ask the user if he or she wants to roll the dices or not. If yes, then the computer will roll a random dices and show its output.
+- Highly useful while playing dice game with your friends.
+- Even more useful if someone doubts the physical dices provided.
+
```
Comment (Abbhiishek) [inline Contributions/Dice Roll game/README.md:9]: Can you add some Screenshots of your working application.

Gifs aur video also works well

**Review by Abbhiishek — CHANGES_REQUESTED** [review]
Some changes are requested before merging the commits.

File: Contributions/Dice Roll game/README.md (hunk 2 above)
Comment (Subha-5) [reply Contributions/Dice Roll game/README.md:9]: [Screencast from 14-10-22 08:21:59 PM IST.webm](https://user-images.githubusercontent.com/97968307/195877249-26933c01-dcd3-4353-a379-ef7f61b4e513.webm)

Comment (Subha-5) [reply Contributions/Dice Roll game/README.md:9]: @Abbhiishek this is a short video on the project

**Comment by Subha-5** [comment]
@Abbhiishek I have made the necessary changes, please review

File: CONTRIBUTORS.json (hunk 1 above)
Comment (Abbhiishek) [reply CONTRIBUTORS.json:316]: It is advised to maintain a single branch for each PR.

File: Contributions/Dice Roll game/README.md (hunk 2 above)
Comment (Abbhiishek) [reply Contributions/Dice Roll game/README.md:9]: Nice demonstration 😮 

- Insert the video in `README.md` file

**Review by Abbhiishek — CHANGES_REQUESTED** [review]
Few more changes ➡️

File: CONTRIBUTORS.json (hunk 1 above)
Comment (Subha-5) [reply CONTRIBUTORS.json:322]: Okay, I will take care of this that I maintain a single branch from next time.

File: Contributions/Dice Roll game/README.md (hunk 2 above)
Comment (Subha-5) [reply Contributions/Dice Roll game/README.md:9]: @Abbhiishek  Inserted the video in Readme. Please take a look.

**Comment by Subha-5** [comment]
@Abbhiishek I have done all the required changes, so please review the PR.

**Review by Abbhiishek — APPROVED** [review]

**Pull request merged**